.travis.yml

```diff
@@ -14,13 +14,26 @@ services:
 
 jobs:
   include:
-      script:
-      - npm ci
-      - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then npm run functional:cron:prod; fi
-      - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then npm run build:prod; fi
-      - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then npm run functional:ci; fi
-      - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then npm run functional:ci:bs; fi
-      - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then npm run test:saucelabs; fi
+      - stage: CRON build
+        if: type == cron
+        script: 
+        - npm ci
+        - npm run functional:cron:prod
+      - stage: PR build
+        if: type == pull_request OR type == push
+        script:
+        - npm ci
+        - npm run build:prod
+        - npm run functional:ci
+        - npm run functional:ci:bs
+        - npm run test:saucelabs
+      - stage: API/Konductor Build
+        if: type == api
+        env:
+          - ALLOY_ENV="prod"
```

Comment:
Hopefully we can eventually remove this from here and Konductor can pass it in the API. Wasn't able to make it work yet.